patcher/HitmanPatcher.Core/AOBScanner.cs

```diff
@@ -96,7 +105,8 @@ public static bool TryGetHitmanVersionByScanning(Process process,
             Note("AuthHeader2", getAuthheadPatches.Result.First()[1]);
             Note("ConfigDomain", getConfigdomainPatches.Result.First()[0]);
             Note("Protocol", getProtocolPatches.Result.First()[0]);
-            Note("DynamicResources", getDynresForceofflinePatches.Result.First()[0]);
+            Note("DynamicResources->ForceOffline", getDynresForceofflinePatches.Result.First()[0]);
```

Comment:
I was going to say something about this potentially being renamed to make it clear that it's changing it to be not mandatory, but on second thought I think this is fine as-is. just noting for posterity

Reply:
Would be better to rename the notes to be the name of the original config commands (where applicable).

Reply:
I have been very tempted to do that (rename it to OnlineResources), but I wanted to go as-is and stay in line with what is already there. 

With this PR out of the way I intend to do a refactor with the .NET Core version and cleanup the code. I'd say that's a good time to also name things the way they should be. There are issues open about the CLI arguments/options being negatively named (noforceoffline), the fact some options should not even be an option, etc.

I therefor propose to leave the renaming out of this PR.

Reply:
SGTM